strtoint.go

```diff
@@ -0,0 +1,36 @@
+package strtoint
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+func MyStrToInt(s string) (result int, err error) {
+	result, err = strconv.Atoi(Filter(s))
```

Comment:
It's better to see an error instead of wrong number  In case of typo, remove ```Filter``` function is not a good idea